{
    "api_version": 7,
    "all_tests_passed": false,
    "user_id": 2702,
    "course": "k2015-cee",
    "exercise_name": "Module_6-Task_1",
    "status": "fail",
    "points": [],
    "processing_time": 6,
    "message_for_paste": "",
    "missing_review_points": [],
    "test_cases":
            [
                {
                    "name": "Test-6.1 test_turn_around",
                    "successful": true,
                    "message": "Passed",
                    "exception": null,
                    "detailed_message": null

                },
                {
                    "name": "Test-6.1 test_is_reversed",
                    "successful": true,
                    "message": "Passed",
                    "exception": null,
                    "detailed_message": null

                },
                {
                    "name": "Test-6.1 test_reversed_words",
                    "successful": true,
                    "message": "Passed",
                    "exception": null,
                    "detailed_message": null

                },
                {
                    "name": "Test-6.1 test_most_common",
                    "successful": false,
                    "message": "Received signal 11 (Segmentation fault)",
                    "exception": null,
                    "detailed_message": null
                }
            ],
    "solution_url": "https://tmc.mooc.fi/hy/exercises/3783/solution",
    "validations": null,
    "valgrind": "==394== Use of uninitialised value of size 8\n==394==    at 0x40325E: most_common (strings.c:171)\n==394==    by 0x4024AC: test_most_common (test_source.c:170)\n==394==    by 0x406290: srunner_run_all (in /tmc/test/test)\n==394==    by 0x4029D2: tmc_run_tests (tmc-check.c:122)\n==394==    by 0x40268D: main (test_source.c:192)\n==394==  Uninitialised value was created by a stack allocation\n==394==    at 0x40339A: findMostCommon (strings.c:205)\n==394== \n==394== Invalid read of size 8\n==394==    at 0x40325E: most_common (strings.c:171)\n==394==    by 0x4024AC: test_most_common (test_source.c:170)\n==394==    by 0x406290: srunner_run_all (in /tmc/test/test)\n==394==    by 0x4029D2: tmc_run_tests (tmc-check.c:122)\n==394==    by 0x40268D: main (test_source.c:192)\n==394==  Address 0x305185064 is not stack'd, malloc'd or (recently) free'd\n==394== \n==394== \n==394== Process terminating with default action of signal 11 (SIGSEGV)\n==394==  Access not within mapped region at address 0x305185064\n==394==    at 0x40325E: most_common (strings.c:171)\n==394==    by 0x4024AC: test_most_common (test_source.c:170)\n==394==    by 0x406290: srunner_run_all (in /tmc/test/test)\n==394==    by 0x4029D2: tmc_run_tests (tmc-check.c:122)\n==394==    by 0x40268D: main (test_source.c:192)\n==394==  If you believe this happened as a result of a stack\n==394==  overflow in your program's main thread (unlikely but\n==394==  possible), you can try to increase the size of the\n==394==  main thread stack using the --main-stacksize= flag.\n==394==  The main thread stack size used in this run was 8388608.\n",
    "reviewed": false,
    "requests_review": false,
    "submitted_at": "2015-05-26T14:16:25.451+03:00"

}